examples/qualcomm/qaihub_scripts/llama/llama2/qaihub_llama2_7b.py

```diff
@@ -230,7 +220,7 @@ def post_process():
     parser.add_argument(
         "--temperature",
         help="sampling temperature for llama2",
-        default=0.8,
+        default=0.0,
```

Comment:
Any specific reason we're using 0 temperature?

Reply:
We change the default to 0 because the output can be more consistent, which is better for testing purposes.